src/process/basic_process.cpp

```diff
@@ -214,10 +214,9 @@ void mp::BasicProcess::setup_child_process()
 void mp::BasicProcess::handle_started()
 {
     pid = process.processId(); // save this, so we know it even after finished
-    const auto& program = qUtf8Printable(process_spec->program());
-
+    const auto program = process_spec->program().toStdString();
```

Comment:
This is ok, but the ref wasn't wrong either: const refs extend the lifetime of temporaries. That's why we can do `f("literal")` when we have `voic f(const string&)`. The same is true for rvalue refs (lvalue refs can't be bound to temporaries). A couple of references: [GotW88](https://herbsutter.com/2008/01/01/gotw-88-a-candidate-for-the-most-important-const/) & [cppref](https://en.cppreference.com/w/cpp/language/reference_initialization#Lifetime_of_a_temporary).

Reply:
Sure, I can (and should) leave the ref to the `std::string` now.  I'll explain below why I removed the `qtf8Printable`s.

Reply:
Ok, fixed this.  Thanks!